src/objectid.ts

```diff
@@ -287,12 +256,7 @@ export class ObjectId {
   /** Returns the generation date (accurate up to the second) that this ID was generated. */
   getTimestamp(): Date {
     const timestamp = new Date();
-    let time: number;
-    if (typeof this.id !== 'string') {
-      time = this.id.readUInt32BE(0);
-    } else {
-      time = this.generationTime;
-    }
+    const time = this.id.readUInt32BE(0, false);
```

Comment:
where are these `false` parameters coming from? I don't see them in the [node docs](https://nodejs.org/api/buffer.html#buffer_buf_readuint32be_offset)

Reply:
Oops legacy node pre 1.0 arguments, removed.